README.md

```diff
@@ -1,3 +1,31 @@
-# radiance ☀️
+# radiance #
 
 Highly experimental Solana playing ground.
+
+## Available tooling ##
+
+- [solrays](cmd/solrays), an RPC proxy that exports call latency of a Solana RPC node.
+
+## Building ##
+
+In order to build the available tooling, the following is required:
+- Go 1.18+
+- Run
+  ```
+  ./generate.sh
+  ```
+
+Building all the tools _should_ be as easy as:
```

Comment:
Why "should"?

Reply:
Some tools may not be as easy, eg tools that depend on CGO and C headers, eg
```
:; go build -o _bin/ github.com/certusone/radiance/cmd/...
# github.com/lucas-clemente/quic-go/internal/qtls
../../go/pkg/mod/github.com/leoluk/quic-go@v0.27.1-0.20220617104859-ebc95102ed1c/internal/qtls/go119.go:6:13: cannot use "The version of quic-go you're using can't be built on Go 1.19 yet. For more details, please see https://github.com/lucas-clemente/quic-go/wiki/quic-go-and-Go-versions." (untyped string constant "The version of quic-go you're using can't be built on Go 1.19 yet. F...) as int value in variable declaration
# github.com/LiamHaworth/go-tproxy
../../go/pkg/mod/github.com/!liam!haworth/go-tproxy@v0.0.0-20190726054950-ef7efd7f24ed/tproxy_udp.go:30:57: undefined: syscall.SOL_IP
../../go/pkg/mod/github.com/!liam!haworth/go-tproxy@v0.0.0-20190726054950-ef7efd7f24ed/tproxy_udp.go:30:73: undefined: syscall.IP_TRANSPARENT
../../go/pkg/mod/github.com/!liam!haworth/go-tproxy@v0.0.0-20190726054950-ef7efd7f24ed/tproxy_udp.go:34:57: undefined: syscall.SOL_IP
../../go/pkg/mod/github.com/!liam!haworth/go-tproxy@v0.0.0-20190726054950-ef7efd7f24ed/tproxy_udp.go:34:73: undefined: syscall.IP_RECVORIGDSTADDR
../../go/pkg/mod/github.com/!liam!haworth/go-tproxy@v0.0.0-20190726054950-ef7efd7f24ed/tproxy_udp.go:61:34: undefined: syscall.SOL_IP
../../go/pkg/mod/github.com/!liam!haworth/go-tproxy@v0.0.0-20190726054950-ef7efd7f24ed/tproxy_udp.go:61:71: undefined: syscall.IP_RECVORIGDSTADDR
../../go/pkg/mod/github.com/!liam!haworth/go-tproxy@v0.0.0-20190726054950-ef7efd7f24ed/tproxy_udp.go:122:57: undefined: syscall.SOL_IP
../../go/pkg/mod/github.com/!liam!haworth/go-tproxy@v0.0.0-20190726054950-ef7efd7f24ed/tproxy_udp.go:122:73: undefined: syscall.IP_TRANSPARENT
# github.com/google/nftables/xt
../../go/pkg/mod/github.com/google/nftables@v0.0.0-20220611213346-a346d51f53b3/xt/info.go:51:18: undefined: unix.NFPROTO_IPV4
../../go/pkg/mod/github.com/google/nftables@v0.0.0-20220611213346-a346d51f53b3/xt/info.go:56:13: undefined: unix.NFPROTO_IPV4
../../go/pkg/mod/github.com/google/nftables@v0.0.0-20220611213346-a346d51f53b3/xt/info.go:62:13: undefined: unix.NFPROTO_IPV6
../../go/pkg/mod/github.com/google/nftables@v0.0.0-20220611213346-a346d51f53b3/xt/info.go:72:13: undefined: unix.NFPROTO_IPV4
../../go/pkg/mod/github.com/google/nftables@v0.0.0-20220611213346-a346d51f53b3/xt/info.go:77:13: undefined: unix.NFPROTO_IPV4
../../go/pkg/mod/github.com/google/nftables@v0.0.0-20220611213346-a346d51f53b3/xt/info.go:83:13: undefined: unix.NFPROTO_IPV6
../../go/pkg/mod/github.com/google/nftables@v0.0.0-20220611213346-a346d51f53b3/xt/util.go:36:12: undefined: unix.NFPROTO_IPV4
../../go/pkg/mod/github.com/google/nftables@v0.0.0-20220611213346-a346d51f53b3/xt/util.go:38:12: undefined: unix.NFPROTO_IPV6
../../go/pkg/mod/github.com/google/nftables@v0.0.0-20220611213346-a346d51f53b3/xt/util.go:52:12: undefined: unix.NFPROTO_IPV4
../../go/pkg/mod/github.com/google/nftables@v0.0.0-20220611213346-a346d51f53b3/xt/util.go:54:12: undefined: unix.NFPROTO_IPV6
../../go/pkg/mod/github.com/google/nftables@v0.0.0-20220611213346-a346d51f53b3/xt/util.go:54:12: too many errors
# github.com/linxGnu/grocksdb
../../go/pkg/mod/github.com/linx!gnu/grocksdb@v1.7.7/array.go:4:11: fatal error: 'rocksdb/c.h' file not found
 #include "rocksdb/c.h"
          ^~~~~~~~~~~~~
1 error generated.
```